modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -106,50 +106,83 @@ definition {
 		}
 	}
 
-	@description = "LPS-180070 - Verify creating a Commerce Product entry triggers an action to add a user"
+	@description = "LPS-181943 - Verify deleting a Commerce Product entry triggers an action to add a Commerce Product Group entry"
 	@priority = 5
-	test CanAddUserAfterCreatingCommerceProductEntry {
+	test CanAddCommerceProductGroupEntryAfterDeletingCommerceProductEntry {
 		property portal.acceptance = "true";
 
-		task ("Given a Commerce Product object") {
+		task ("Given a minium site and Commerce Product object") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Product Management",
+				panel = "Commerce",
+				portlet = "Products");
+
+			CPCommerceCatalog.newProductsEntry(
+				productCatalog = "Minium",
+				productName = "Simple T-Shirt",
+				productType = "Simple");
+
+			ApplicationsMenu.gotoSite(site = "Minium");
+
+			CommerceAcceleratorsInitializer.viewMiniumCardName(productName = "Simple T-Shirt");
+
+			CommerceAcceleratorsInitializer.gotoMiniumProductDetailsPage(productName = "Simple T-Shirt");
+
+			FrontStore.assertProductDetails(
+				productFullDescription = "Simple T-Shirt Full Description",
+				productName = "Simple T-Shirt",
+				productShortDescription = "Simple T-Shirt Short Description");
+		}
+
+		task ("And given an action to add a Commerce Product Group entry after deleting a Commerce Product entry") {
```

Comment:
Hey @andreefarias I think something went wrong while cherry picking the commit. Can you fix this by cherry picking https://github.com/liferay-bpm-qa/liferay-portal/pull/721/commits/3a1d8d823fd27b9d42f185896beb3e196f76967a again?

Reply:
From reviewing the test from the original commit, the test is fine but we should remove the extra assertion to check the status of the action.

Reply:
We also shouldn't be navigating to the minium site to assert the entry was created in the given. There's another example of this in the next test.

Reply:
Sure